Directory.Packages.props

```diff
@@ -12,18 +12,18 @@
   <ItemGroup>
     <!-- AWS SDK for .NET dependencies -->
     <PackageVersion Include="AspNetCore.HealthChecks.Oracle" Version="8.0.1" />
-    <PackageVersion Include="AWSSDK.CloudFormation" Version="3.7.400.15" />
-    <PackageVersion Include="AWSSDK.SQS" Version="3.7.400.15" />
-    <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.400.15" />
-    <PackageVersion Include="AWSSDK.Core" Version="3.7.400.15" />
+    <PackageVersion Include="AWSSDK.CloudFormation" Version="3.7.400.26" />
+    <PackageVersion Include="AWSSDK.SQS" Version="3.7.400.26" />
+    <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.400.26" />
+    <PackageVersion Include="AWSSDK.Core" Version="3.7.400.26" />
     <PackageVersion Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.301" />
     <PackageVersion Include="AWS.Messaging" Version="0.9.2" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.AWS" Version="1.1.0-beta.5" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.AWS" Version="1.1.0-beta.6" />
     <PackageVersion Include="OpenTelemetry.Extensions.AWS" Version="1.3.0-beta.1" />
     <!-- AWS CDK dependencies -->
-    <PackageVersion Include="Amazon.CDK.Lib" Version="2.158.0" />
+    <PackageVersion Include="Amazon.CDK.Lib" Version="2.160.0" />
     <!-- Azure SDK for .NET dependencies -->
-    <PackageVersion Include="Azure.AI.OpenAI" Version="2.0.0-beta.6" />
+    <PackageVersion Include="Azure.AI.OpenAI" Version="2.0.0" />
```

Comment:
This is taken care of in [Update OpenAI packages (dotnet/aspire#6060)](https://github.com/dotnet/aspire/pull/6060). It can be reverted here. Same for OpenAI.

That way you don't need to fix the build / config json / tests.